packages/react-components/src/components/MessageThread.test.tsx

```diff
@@ -220,7 +220,7 @@ describe.only('Message should display image and attachment correctly', () => {
       onFetchAttachmentCount++;
       return [
         {
-          blobUrl: attachment.previewUrl ?? ''
+          blobUrl: (attachment as TeamsInteropFileMetadata).previewUrl ?? ''

```

Comment:
nit: avoid `as` as much as possible as it circumvents the type guarding. Instead if you use your helper function `isTeamsInteropFileMetadata` it will strongly the type the attachment:

```
if (isTeamsInteropFileMetadata(attachement) {
  // you can now use attachment.previewUrl without typecasting
}
```

Reply:
Updated to se the helper where ever possible